setup.rst

```diff
@@ -447,8 +459,8 @@ For editors and tools which the core developers have felt some special comment
 is needed for coding *in* Python, see :ref:`resources`.
 
 
-Directory Structure
-===================
+Explore CPython's directory structure
+=====================================
```

Comment:
As above, I don't feel like "explore" adds anything -- it just makes the title longer.
I'm fine with leaving "CPython's" though.